src/App/App.module.css

```diff
@@ -54,3 +55,71 @@ a {
   opacity: .3;
   transition: ease .1s;
 }
+
+.promo {
+  display: flex;
+  position: absolute;
+  bottom: 10px;
+  right: 10px;
+  font-family: 'Helvetica', sans-serif;
+  font-style: italic;
+}
+
+.promo > img {
+  margin-left: 5px;
+}
+
+@media (max-width: 1024px) {
+  .promo {
+    display: none;
+  }
+}
+
+@media (max-width: 610px) {
+  .wraper {
+    flex-direction: column;
+  }
+
+  .menu {
+    flex-direction: row;
+    width: 400px;
+    margin: 0 auto;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .content {
+    margin: 0 auto;
+  }
+
+  .item {
+    font-size: 1.5em;
+    margin-bottom: 0;
+  }
+
+  .menu > a {
+  border: 3px solid #3f51b5;
+  border-radius: 15px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/dotodo/pullRequest?prid=6099781)